packages/trader/src/Modules/CFD/Containers/cfd-password-modal.jsx

```diff
@@ -30,7 +30,7 @@ import { localize, Localize } from '@deriv/translations';
 import SuccessDialog from 'App/Containers/Modals/success-dialog.jsx';
 import 'Sass/app/modules/mt5/cfd.scss';
 import { connect } from 'Stores/connect';
-import ChangePasswordConfirmation from './cfd-change-password-confirmation.jsx';
+import ChangePasswordConfirmation from './cfd-change-password-confirmation.tsx';
```

Comment:
```suggestion
import ChangePasswordConfirmation from './cfd-change-password-confirmation';
```

Since we will be gradually upgrading to `tsx` renaming import extensions would be quite the chore. Cant we do this type of import instead. That way it would be generic and cater for both `tsx` and `jsx`

Reply:
All of the changes are made. I'll avoid React.FC typing since It provides typechecking and autocomplete for static properties like displayName, propTypes, and defaultProps and there are some issues using defaultProps with React.FunctionComponent.